telemetry/definitions/commonDefinitions.json

```diff
@@ -3808,6 +3808,20 @@
                 }
             ]
         },
+        {
+            "name": "codeTransform_mvnCommandsExecutionComplete",
```

Comment:
- What happens if you add `gradle` support? Are you going to create a bunch of `gradle` metrics? 
    - Why not store the "build tool" as a field in the metric instead?
- `ExecutionComplete` is another thing that just doesn't need to be spelled out in the metric name. Just call it `build`.

Based on the above, this metric could be named `codeTransform_build`, and it could have a `buildTool` field.